templates/admin/streams/edit.html.twig

```diff
@@ -0,0 +1,44 @@
+{% extends 'admin/admin.html.twig' %}
+
+{% block content %}
+
+	<div class="row">
+		<div class="col s12">
+			<h2>{{ stream.streamName }}
+				<h3>{{ stream.activityName }}</h3>
+
+				{% if stream.picture is not null %}
+					<img src="{{ asset('uploads/partners/' ~ stream.picture) }}" alt="{{ stream.streamName }}" class="right">
```

Comment:
```suggestion
					<img src="{{ asset('uploads/streamers/' ~ stream.picture) }}" alt="{{ stream.streamName }}" class="right">
```

Since it's for streamers, we should have the image on this folder instead or partners.